src/blurredLocation.js

```diff
@@ -10,6 +10,14 @@ BlurredLocation = function BlurredLocation(options) {
 
   options.gridSystem = options.gridSystem || require('./core/gridSystem.js');
   options.Interface = options.Interface || require('./ui/Interface.js');
+  options.onDrag = options.onDrag || function onDrag() {
+    function changeVal(result) {
+      $("#location").val(result.results[0].formatted_address);
+    }
+    options.map.on('moveend', function() {
```

Comment:
Then this should just be outside the function, as part of the setup sequence of the library. Maybe run within `Interface = options.Interface(InterfaceOptions);` -- make sense? Then it will attach to the onDrag method, but when a user of the library assigns something new to `onDrag`, like this:

```js
blurredLocation.options.onDrag = function myNewOnDrag() {
  // whatever they want to happen upon a drag
}
```

it'll run theirs instead of ours.